public/data/Akshay-bendadi.json

```
@@ -18,6 +18,11 @@
       "name": "Follow me on LinkedIn",
       "url": "https://www.linkedin.com/in/akshay-bendadi/",
       "icon": "linkedin"
+    },
+    {
+      "name": "Follow me on hashnode",
```

I recommend capitalizing the `h` here. What do you think?
```suggestion
      "name": "Follow me on Hashnode",
```